proposals/bring_your_own_argo_workflows/byoaw_context.md

```
@@ -0,0 +1,155 @@
+  ...
+  ```
+
+With that said, for RHOAI installations with a large number of DSPAs it would be unsustainable to require editing every DSPA individually.  A global toggle mechanism must be implemented instead - one which would remove the Workflow Controller from ALL managed DSPAs.  This would be set in the DataScienceCluster CR (see example below) and would involve coordination with the Platform dev team for implementation.  Given that, documentation in the DSPA CRD will need to be added to notify users that it is an unsupported configuration to have individual WCs disabled if a user is providing their own Argo WorkflowController, and that the field is for development purposes only.
+
+Example DataScienceCluster with WorkflowControllers globally disabled:
+```---
+kind: DataScienceCluster
+...
+spec:
+  components:
+	...
```

markdownlint-cli2 (0.17.2)

Hard tabs
Column: 1

(MD010, no-hard-tabs)